ESBreakerCLI/ParseTextStrategies.cs

```diff
@@ -882,5 +884,59 @@ public static JsonArrayCollection RashArtsExplain(object source, JsonArrayCollec
             }
             return data;
         }
-    }
+		public static JsonArrayCollection ChipAwakeningExplainTokens(object source, JsonArrayCollection existingData, bool saveJson)
+		{
+			JsonArrayCollection data = new JsonArrayCollection();
+
+			Contents.Text.ChipAwakeningExplainTokens.Format tokensTitle = (Contents.Text.ChipAwakeningExplainTokens.Format)source;
+			if (tokensTitle != null)
+			{
+
+				foreach (Contents.Text.ChipAwakeningExplainTokens.Information information in tokensTitle.Information)
+				{
+					int i = 0;
+					for (int idx = 0; idx < information.Tokens.Length; idx++)
+					{
+						JsonObjectCollection savedItem = default(JsonObjectCollection);
+						var param = information.Tokens[idx];
+						JsonObjectCollection item = new JsonObjectCollection();
+
+						if (existingData != null)
+						{
+							int Count = existingData.Count;
+							for (int idy = 0; idy < Count; idy++)
+							{
+								if (existingData[idy] != null &&
+									existingData[idy]["jp_token"] == param)
+								{
+									savedItem = (JsonObjectCollection)existingData[idy];
+									existingData.RemoveAt(idy);
+									break;
+								}
+							}
+						}
+
+						
```

Comment:
There is empty space here
![image](https://user-images.githubusercontent.com/188068/59859940-b335f980-934b-11e9-8bee-73a7e3bb9aec.png)